src/commands/bindings.rs

```diff
@@ -87,7 +91,55 @@ The expected file is {wit_path:?}"#
                 println!("Bindings generated for Rust in {0}. You need to add the `wit-bindgen` crate to your Rust Spin app - e.g., `cargo add wit-bindgen`", self.output.to_str().expect("Failed to parse output path"));
             }
             BindingsLanguage::Ts => {
-                todo!("generate ts")
+                let imported_interfaces = get_imported_interfaces(&resolve, world_id);
+
+                let files = generate_types(
+                    // This name does not matter as we are not going to use it
+                    "test".to_string(),
+                    resolve,
+                    world_id,
+                    TranspileOpts {
+                        name: self.output.to_str().unwrap().to_string(),
+                        no_typescript: false,
+                        instantiation: None,
+                        import_bindings: None,
+                        map: None,
+                        no_nodejs_compat: false,
+                        base64_cutoff: 0,
+                        tla_compat: false,
+                        valid_lifting_optimization: false,
+                        tracing: false,
+                        no_namespaced_exports: false,
+                        multi_memory: true,
+                        guest: true,
+                    },
+                )?;
+
+                for (name, contents) in files.iter() {
+                    let output_path = self.output.join(name);
+                    if !output_path.to_str().unwrap().contains("/interfaces/") {
```

Comment:
This seems like it include _all_ files if `self.output` contains `/interfaces/`.  Is that intentional?  Or would it be better to filter on `name` before constructing `output_path`?

Are there any package / world / interface names that could make a naive `contains` check go awry?